src/Effects/Sympathetic.cpp

```diff
@@ -213,20 +230,56 @@ void Sympathetic::calcFreqs()
 
 }
 
+void Sympathetic::calcFreqsPiano()
+{
+    const float unison_spread_semicent = powf(Punison_frequency_spread / 63.5f, 2.0f) * 25.0f;
+    const float unison_real_spread_up = powf(2.0f, (unison_spread_semicent * 0.5f) / 1200.0f);
+    const float unison_real_spread_down = 1.0f/unison_real_spread_up;
+
+    for(unsigned int i = 0; i < Punison_size*Pstrings; i+=Punison_size)
+    {
+        const float centerFreq = powf(2.0f, (float)i / 36.0f) * baseFreq;
+        const unsigned int stringchoir_size = 
+			i>num_single_strings ? (i>Pstrings-num_triple_strings ? 3 : 2) :1;
```

Comment:
I thought pianos have 3 strings for the *lowest* keys, not for the highest ones?

Reply:
take a second look ;-)
the lower strings have more mass. therefor they can store more energy. to compensate that higher notes get more strings.

Reply:
Oh, you're right, I got this wrong way round :+1: